### PR TITLE
FOGL-5253

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

**Review by YashTatkondawar — DISMISSED** [review]

File: docker/Dockerfile [hunk 1]
```diff
@@ -30,7 +26,23 @@ VOLUME /usr/local/fledge/data
 VOLUME /usr/local/fledge
 VOLUME /fledge-core/fledge
 
-EXPOSE 8081 1995
+EXPOSE 8081 1995 80
 
-CMD ["bash", "/fledge-core/start.sh"]
 
+RUN wget -q -O - http://archives.fledge-iot.org/KEY.gpg | sudo apt-key add -
+RUN echo "deb http://archives.fledge-iot.org/nightly/ubuntu1804/x86_64/ / " > /etc/apt/sources.list.d/fledge.list
```
Comment (praveen-garg) [inline docker/Dockerfile:33]: Do we have WL handling for this?

File: docker/latest/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="info@dianomic.com"
+
+ENV ARCHITECTURE="x86_64"
+ENV OS="ubuntu1804"
+
+ARG PKG_VERSION="latest"
+
+ENV FLEDGE_PKGS_ARCHIVES_LINK="http://archives.fledge-iot.org/${PKG_VERSION}/${OS}/${ARCHITECTURE}/"
```
Comment (praveen-garg) [inline docker/latest/Dockerfile:10]: Do we have WL handling for this?

**Review by brycekroencke — APPROVED** [review]
I was able to successfully use the Dockerfile to setup the OSDU use case

File: docker/Dockerfile (hunk 1 above)
Comment (YashTatkondawar) [reply docker/Dockerfile:33]: Yes, it is handled in white-labeling scripts.

File: docker/latest/Dockerfile (hunk 2 above)
Comment (YashTatkondawar) [reply docker/latest/Dockerfile:10]: Yes, it is handled in white-labeling scripts.

**Review by YashTatkondawar — APPROVED** [review]

**Pull request merged**